docs/diagrams/ScheduleSequenceDiagram2.puml

```diff
@@ -4,7 +4,7 @@ skinparam defaultFontSize 20
 
 box Logic LOGIC_COLOR_T1
 
-group ref [execute ScheduleFriendCommand]
+group sd [execute ScheduleFriendCommand]
```

Comment:
sd should be inside the [ ]
![image](https://user-images.githubusercontent.com/50147457/140679429-fcb01523-e548-4242-a243-dd2852e7576a.png)